lib/dao.js

```diff
@@ -1558,65 +1559,64 @@ DataAccessObject.find = function find(query, options, cb) {
           queryGeo(ctx.query);
         });
       }
+      // already handled
+      return cb.promise;
```

Comment:
Sorry I am a little bit confused, my understanding is the new code returns `cb.promise` outside of the `if...else` statement, so what is the difference between returning it outside and inside?

Reply:
Actually the `cb.promise` call is in the same spot as before (inside the `if(supportsGeo)...else if (query.where)` block). It's hard to see it from the changes (Disabling white-space changes in the Diff settings helps)

Reply:
Thank you I see, much better after disabling white-space!